wire-ios/Wire-iOS/Sources/LaunchSequenceOperation.swift

```diff
@@ -86,7 +86,7 @@ final class TrackingOperation: LaunchSequenceOperation {
             .arguments.contains(AnalyticsProviderFactory.ZMConsoleAnalyticsArgumentKey)
 
         AnalyticsProviderFactory.shared.useConsoleAnalytics = containsConsoleAnalytics
-        Analytics.shared = Analytics(optedOut: TrackingManager.shared.disableAnalyticsSharing)
+       // Analytics.shared = Analytics(optedOut: TrackingManager.shared.disableAnalyticsSharing)
```

Comment:
To remove?

Reply:
Yep, removed it here: https://github.com/wireapp/wire-ios/commit/66eb745653a5cab5098e1264ebbb6402d69820dd